parlai/tasks/task_list.py

```diff
@@ -171,25 +171,32 @@
         "tags": [ "all",  "QA" ],
         "description": "Open domain QA from Wikipedia dataset from Yang et al. '15. Link: https://www.microsoft.com/en-us/research/publication/wikiqa-a-challenge-dataset-for-open-domain-question-answering/"
     },
-	{
-		"id": "VQAv1",
-		"display_name": "VQAv1",
-		"task": "vqa_v1",
-		"tags": [ "all", "Visual"],
-		"description": "Open-ended question answering about visual content. From Agrawal et al. '15. Link: https://arxiv.org/abs/1505.00468"
-	},
-	{
-		"id": "VQAv2",
-		"display_name": "VQAv2",
-		"task": "vqa_v2",
-		"tags": [ "all", "Visual"],
-		"description": "Bigger, more balanced version of the original VQA dataset. From Goyal et al. '16. Link: https://arxiv.org/abs/1612.00837"
-	},
-	{
-		"id": "VisDial",
-		"display_name": "VisDial",
-		"task": "visdial",
-		"tags": [ "all", "Visual"],
-		"description": "Task which requires agents to hold a meaningful dialog about visual content. From Das et al. '16. Link: https://arxiv.org/abs/1611.08669"
-	},
+    {
+            "id": "VQAv1",
+            "display_name": "VQAv1",
+            "task": "vqa_v1",
+            "tags": [ "all", "Visual"],
+            "description": "Open-ended question answering about visual content. From Agrawal et al. '15. Link: https://arxiv.org/abs/1505.00468"
+    },
+    {
+            "id": "VQAv2",
+            "display_name": "VQAv2",
+            "task": "vqa_v2",
+            "tags": [ "all", "Visual"],
+            "description": "Bigger, more balanced version of the original VQA dataset. From Goyal et al. '16. Link: https://arxiv.org/abs/1612.00837"
+    },
+    {
+            "id": "VisDial",
+            "display_name": "VisDial",
+            "task": "visdial",
+            "tags": [ "all", "Visual"],
+            "description": "Task which requires agents to hold a meaningful dialog about visual content. From Das et al. '16. Link: https://arxiv.org/abs/1611.08669"
+    },
+    {
+            "id": "MNIST_QA",
+            "display_name": "MNIST_QA",
+            "task": "mnist_qa",
+            "tags": [ "all", "QA", "Visual"],
```

Comment:
one last thing: since this **requires** the image to answer the question, let's remove the QA tag here